midi-to-part-mp3s

```diff
@@ -26,6 +27,34 @@ class Part:
     def mp3_filepath(self):
         return self.midi_filepath.replace(".midi", ".mp3")
 
+def check_format(file_path: str) -> str:
+  """If the format is midi, proceed. If the format is MusicXML, convert to midi.
+  
+  Arguments:
+      file_path {[str]} -- path of the file to convert
+  
+  Raises:
+      NameError: Raised if trying to convert a format not supported by the script
+  
+  Returns:
+      str -- midi file name (original or converted midi file)
+  """
+  if file_path.endswith('.mid') or file_path.endswith('.midi'):
+    return file_path
+  elif file_path.endswith('.mxl'):
+    converted_file_path = output_directory + '/temp.midi'
+    c: music21.converter = music21.converter.Converter()
```

Comment:
Again, another dumb question: I'm not familiar with this syntax where a variable is placed before a colon. Can you send me a link? 

Probably related, though I don't see where this `c` variable gets consumed.

Reply:
It's also a type hint, but you are right, I am not using the variable, probably forgot to delete the line. Should also be music21.converter.Converter: c